remoteappmanager/db/orm.py

```diff
@@ -0,0 +1,142 @@
+import contextlib
+
+from sqlalchemy import Column, Integer, Boolean, Unicode, ForeignKey
+from sqlalchemy.exc import OperationalError
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from sqlalchemy import create_engine
+
+from remoteappmanager.logging.logging_mixin import LoggingMixin
+
+Base = declarative_base()
+
+
+class UserTeam(Base):
+    """ The user (n <-> n) team association table """
+    __tablename__ = "user_team"
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, ForeignKey('user.id'))
+    team_id = Column(Integer, ForeignKey('team.id'))
```

Comment:
Just checked on my local machine, if you add primary key to all of the columns (`id`, `user_id`, `team_id`), the SQL table would have a primary key `PRIMARY KEY (id, user_id, team_id)`
So there is no need for the `id` column.